sale_order_incoterm_place/models/sale_order.py

```diff
@@ -0,0 +1,10 @@
+# Copyright 2020 Quartile Limited
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    destination_place = fields.Char("Place of Destination")
```

Comment:
Could you be a little be more precise here. 

And if it is linked to incoterm (cfr the view you modified), I would have done incoterm_destination_location instead.

Reply:
Thanks for the comment, I think we might follow @yostashiro comment to rename it to `incoterm_place`.